draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -35,10 +35,25 @@ $row-height: 60px;
   text-overflow: ellipsis;
   overflow-x: hidden;
   white-space: nowrap;
-  padding: 0 $ca-grid;
-  height: $ca-grid * 1.5;
+  padding: 8px $ca-grid;
   display: flex;
   align-items: center;
+  text-align: left;
+  justify-content: flex-start;
+}
+
+.headerRowCellWrap {
+  white-space: normal;
+}
+
+.headerRowCellAlignCenter {
+  text-align: center;
+  justify-content: center;
+}
+
+.headerRowCellAlignEnd {
+  text-align: right;
```

Comment:
Just confirming, does this work as expected for RTL languages?

Reply:
Depends what you're expecting. The easiest way to check (or at least what I do) is to add `#root { direction: rtl; }` in the dev console. There look to be some alignment issues: 
![Screen Shot 2020-11-04 at 10 21 18 am](https://user-images.githubusercontent.com/1621182/98051251-912d1080-1e87-11eb-8e57-0b76f183ad1a.png)

Reply:
Use `text-align: right / left` when you want to ignore the user's direction preferences or in situations where alignment is independant or direction, and `text-align: start / end` when you want to change alignment based on direction

Reply:
Yeah I never used `start / end` that because of IE11, but now that you mention it, the IE11 styling doesn't need to be perfect. I've fixed that up now.